portfolio/target/appengine-staging/WEB-INF/appengine-generated/app.yaml

```diff
@@ -0,0 +1,45 @@
+runtime: java8
```

Comment:
everything in appengine-staging should probably be in the .gitignore file as well

Reply:
would i write: target/portfolio/appengine-staging , inside of the gitignore file?

Reply:
You can use pattern matching to match everything inside that directory, something like:
```
portfolio/target/appengine-staging/**
```
Here's more information: https://git-scm.com/docs/gitignore